components/Carosel/carousel2.js

```diff
@@ -40,8 +40,7 @@ const T_shirt_line = () => {
         );
       }
         const tShirtArray = tshirts.data.tshirt;
-        console.log(tShirtArray);
-        const starRating = starPower();
+        // console.log(tShirtArray);
```

Comment:
- Got rid of old star rating import, and added the one I made that works.
- Commented out your log to not blow up the console